### DISCUSSION TITLE
[9 Settembre 2022] - Modifica CPU

### DISCUSSION THREAD

**Discussion by rimaout** [CS-Swap/Architettura-degli-Elaboratori#30] [Modifica Architettura]
### Traccia Esercizio

<img width="986" alt="image" src="https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/116072651/44ec55f9-6b0c-41aa-89c3-b56af69a2a5f">
<img width="986" alt="image" src="https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/116072651/158d0124-9882-47ab-8a3b-21968590f5fe">

<details><summary>Soluzione Prof</summary><p>
![image](https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/116072651/c9e6dfc0-bfc3-4c21-ae4f-8e2f19827e64)
<img width="816" alt="image" src="https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/116072651/d14138f6-aabb-4af6-837f-f12644b7fbc6">

 </p></details>

**Comment by notedo** [comment]
![ESERCIZI - Architettura-59](https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/110790224/eb38e94e-0573-41da-9c10-2dd4da61f0af)
![IMG_0493](https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/110790224/1e667946-14a1-435b-83ee-01af88f221d0)
